app/Actions/TwoFactorAuth/VerifyTwoFactorCode.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace App\Actions\TwoFactorAuth;
+
+use PragmaRX\Google2FA\Google2FA;
+
+class VerifyTwoFactorCode
+{
+    /**
+     * Verify a two-factor authentication code.
+     *
+     * @param  string  $secret The decrypted secret key
+     * @param  string  $code The code to verify
+     * @return bool
+     */
+    public function __invoke(string $secret, string $code): bool
+    {
```

Comment:
```suggestion
    public function __invoke(
        #[\SensitiveParameter] string $secret, 
        #[\SensitiveParameter] string $code
    ): bool {
```

Reply:
Great call! I wasn't aware of that attribute. Thanks!